File: src/main/java/me/leodev/trollingfreedomreborn/commands/NickWithoutEss.java
package me.leodev.trollingfreedomreborn.commands;

import org.bukkit.Bukkit;
import org.bukkit.entity.Player;
import org.bukkit.event.Listener;

public class NickWithoutEss implements Listener {

    public void NickName(final Player p) {

        if (Bukkit.getServer().getPluginManager().getPlugin("Essentials") != null) {
            Nick eighteen = new Nick();
            eighteen.NickName(p);
        } else {
        }
    }

    public void UnNick(Player p) {
        if (Bukkit.getServer().getPluginManager().getPlugin("Essentials") != null) {
            Nick eighteen = new Nick();
            eighteen.UnNick(p);
        } else {
        }
    }
}

File: src/main/java/com/leomadrassi/trollingfreedomreborn/trolls/chat/NickWithoutEss.kt
package com.leomadrassi.trollingfreedomreborn.trolls.chat

import org.bukkit.Bukkit
import org.bukkit.entity.Player
import org.bukkit.event.Listener

class NickWithoutEss : Listener {
    fun NickName(p: Player) {
        if (Bukkit.getServer().pluginManager.getPlugin("Essentials") != null) {
            val eighteen = Nick()
            eighteen.NickName(p)
        }
    }

    fun UnNick(p: Player) {
        if (Bukkit.getServer().pluginManager.getPlugin("Essentials") != null) {
            val eighteen = Nick()
            eighteen.UnNick(p)
        }
    }
}
